Update Symlink infos (#550)
iterate-ch/docs: protocols/sftp/index.md

Symbolic links only work within Mountain Duck if the target of the symbolic link is available within the mounted path.

````{tabs}
```{group-tab} macOS

> Symlinks can only be displayed as such if the target is included in the mount, otherwise they are displayed and function as normal folders.

Reviewer: ```suggestion
Symlinks can only be displayed as such if the target is included in the mount, otherwise they are displayed and function as a regular file or folder.
```

```
```{group-tab} Windows

> Symlink is displayed and treated as a normal folder, regardless of whether the target is included in the mount or not.

Reviewer: ```suggestion
Symlink is displayed and treated as a regular file or folder.
```
